tutorials/schedule-dataflow-jobs-with-cloud-scheduler/scheduler-dataflow-demo/cloudbuild.yaml

```diff
@@ -0,0 +1,32 @@
+# The PROJECT_ID is a cloud build env variable
```

Comment:
If the intention is for users to use Cloud Build, the steps should be documented.

Reply:
update the doc to add the instruction of using `gcloud` to submit the cloudbuilt job.